config/allscripts.yml

```diff
@@ -11,11 +11,12 @@ auth:
   scope: launch/patient patient/*.read offline_access
   confidential_client: True
   steps:
-    - find: "[translate='DEFAULT_LOGIN_BUTTON']"
-    - click
-    - find: "#UserName"
-    - type: "s4s_6-22-16"
-    - find: "#Password"
-    - type: "s4s!2345"
-    - find: "[translate='Login_LogIn']"
-    - click
+    - element: "[translate='DEFAULT_LOGIN_BUTTON']"
```

Comment:
Hopefully this isn't too much bikeshedding, but I hope this is a more intuitive step configuration format. Each step selects an element, and then performs an action on it. The action is a "key" in the yaml dictionary, so it can take any parameters it needs. Currently defined actions are `send_keys` and `click`. Steps can also be marked as optional, and if the element can't be found, they'll be silently skipped.